docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3"
+
+services:
+
+  vsftpd:
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      args:
+        USER_ID: 1000
+        GROUP_ID: 1000
```

Comment:
May I ask for more context on these values?

Reply:
What exactly is unclear here?

Reply:
@fauria Uid `1000` and gid `1000` is the common values of non-root user. By `USER_ID` and `GROUP_ID` arguments we can control an owner of uploaded files. I don't know how to do it by any other way.

Reply:
@mikeevstropov it can be achieved via environment variables